Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+/// Contains the description of a dependency that can be installed using Swift Package Manager.
+public struct SwiftPackageManagerDependencies: Codable, Equatable {
+    /// List of packages that will be installed using Swift Package Manager.
+    public let packages: [Package]
+
+    /// Initializes a new `SwiftPackageManagerDependencies` instance.
+    /// - Parameter packages: List of packages that will be installed using Swift Package Manager.
+    init(packages: [Package]) {
```

Comment:
why is this necessary? I feel like this should be automatically synthesized by the compiler

Reply:
Right! Im removing `init` from `ProjectDescription.CarthageDependencies` as well.